src/pulse3D/excel_writer.py

```diff
@@ -313,6 +316,7 @@ def write_xlsx(
 
     log.info("Computing data metrics for each well.")
 
+    max_activeTwitchForce = 0
```

Comment:
Use `snake_case` not `camelCase`. You can also just make this `max_force_of_recording` to make it more clear what's included in this max